### DISCUSSION TITLE
Opening multiple inventories.

### DISCUSSION THREAD

**Discussion by Schrodinger71** [space-wizards/space-station-14#23328] [Feature Request]
Is it possible to somehow make it possible to open several bags at once?

**Comment by EmoGarbage404** [comment]
WYCI. The general consensus is that having an unlimited number of containers isnt really wanted. Any solution involving a limited number would have to be designed to still be UX friendly when opening a bunch of containers quickly.

**Reply by Schrodinger71** [reply]
Well, what if you set a limit?

**Comment by IamVelcroboy** [comment]
No reason you shouldn't be able to have a belt and a bag open at the same time. Or a belt, and a bag, and any number of things on the floor within reach.

**Reply by Schrodinger71** [reply]
This is not possible with the new grid system

**Reply by IamVelcroboy** [reply]
It's not currently coded that way. But it's certainly possible.

**Reply by Schrodinger71** [reply]
It would just be extremely interesting how this could be fixed.
